libs/src/services/ethContracts/audiusTokenClient.js

```diff
@@ -12,7 +12,7 @@ class AudiusTokenClient {
 
   async balanceOf (account) {
     const balance = await this.AudiusTokenContract.methods.balanceOf(account).call()
-    return parseInt(balance, 10)
+    return this.web3.utils.toBN(balance)
```

Comment:
i think this might already be a BN

Reply:
string :'/

Reply:
ah i see, the issue here is the use of `.call()` - i am almost positive that balanceOf returns BN if invoked directly